scripts/masternode.js

```diff
@@ -1,15 +1,14 @@
 "use strict";
 
+/**
+    * Construct a masternode
```

Comment:
I think the first parameter is required for deconstructed objects? Something like
`@param {Object} masternode - Object to be deconstructed`

Reply:
Previously the JSDoc was not being parsed properly (by IntelliSense), I'm not exactly sure why, but removing the masternode object itself allowed the full doc to render fine.

It works best with flat parameters afaik (but would mean we can't as easily serialise from JSON to Class automatically)